### ISSUE TITLE
Core file translation not working correctly

### ISSUE THREAD

**Issue by BernhardBaumrock** [SkyLundy/Fluency#11]
Hey @SkyLundy 

I wanted to translate missing core translations and make a PR for @dotnetic but it does not translate:

<img width="930" alt="image" src="https://github.com/user-attachments/assets/2dc85a60-cfa0-4ec0-86bd-988424b3f4c2" />

I have the default language translated to german and the second language is english. I guess that's why it gets confused. But I have correctly mapped the default language to english in the module settings.

It does not translate both using the "translate to all" and "translate" (single field) buttons.

**Comment by SkyLundy** [comment]
@BernhardBaumrock Checking this out.

**Comment by SkyLundy** [comment]
@BernhardBaumrock I just pushed an update to development that adds a source language select field to language translator pages. This should allow you to select any language as your default language while selecting a source language that matches the content you are translating. Any language configured in Fluency can be selected

![image](https://github.com/user-attachments/assets/7766e73e-95d5-4650-b552-8f1bb4b20f09)

Let me know if this update solves the problem!

There are other minor changes that were being held off until I could put a bigger release together. One worth mentioning is that Fluency will now cache translations using `WireCache::expireNever`. Clearing translations in cache is now done manually in the module config.

**Comment by BernhardBaumrock** [comment]
Hi @SkyLundy thank you! Could you please change (or add an option) the "translate all" to "translate all missing"?

At the moment when I hit translate all it will overwrite all existing translations, which might not be what I want as some translations might have been manually corrected.

Ok... I could click translate on all missing... But if it is not too much work I think having that option would be great! Thx!

**Comment by SkyLundy** [comment]
@BernhardBaumrock Exploring all ideas- would it work if the "Translate All" button followed the UI, so if "Only show blocks that are not yet translated" is checked, it will only translate any empty fields that are shown? Or would it just be better to have a separate button? 

I'm not opposed to a second button, just thought about whether the translation feature should follow the UI. I never thought about it translating over existing fields, that was a good catch.

**Comment by BernhardBaumrock** [comment]
@SkyLundy good idea! I think there should be at least a note that both fields/settings are connected, otherwise it might not be obvious. Or just change the label to "Click To Translate All Shown Blocks", then it should be obvious, as it's the same wording as the checkbox label below.

**Comment by SkyLundy** [comment]
@BernhardBaumrock Going to get to this as soon as I can. Holidays were pretty hectic haha.

**Comment by BernhardBaumrock** [comment]
@SkyLundy no hurry at all :) Take your time and all the best for 2025 :)
